repositories.bzl

```diff
@@ -166,9 +159,69 @@ def rules_go():
         sha256 = "f04d2373bcaf8aa09bccb08a98a57e721306c8f6043a2a0ee610fd6853dcde3d",
     )
 
+
+def org_golang_x_tools():
+    maybe(
+        http_archive,
+        name = "org_golang_x_tools",
+        # master^1, as of 2018-11-02 (master is currently broken)
+        urls = ["https://codeload.github.com/golang/tools/zip/92b943e6bff73e0dfe9e975d94043d8f31067b06"],
+        strip_prefix = "tools-92b943e6bff73e0dfe9e975d94043d8f31067b06",
+        type = "zip",
+        patches = [
+            "@io_bazel_rules_go//third_party:org_golang_x_tools-gazelle.patch",
+            "@io_bazel_rules_go//third_party:org_golang_x_tools-extras.patch",
+        ],
+        patch_args = ["-p1"],
+        # gazelle args: -go_prefix golang.org/x/tools
+    )
+
+def org_golang_x_sys():
+    maybe(
+        git_repository,
+        name = "org_golang_x_sys",
+        remote = "https://github.com/golang/sys",
+        commit = "e4b3c5e9061176387e7cea65e4dc5853801f3fb7",  # master as of 2018-09-28
+        patches = ["@io_bazel_rules_go//third_party:org_golang_x_sys-gazelle.patch"],
+        patch_args = ["-p1"],
+        # gazelle args: -go_prefix golang.org/x/sys
+    )
+
+def io_bazel_rules_go():
+    org_golang_x_tools()
+    org_golang_x_sys()
+    http_archive(
+        name = "io_bazel_rules_go",
+        sha256 = "7be7dc01f1e0afdba6c8eb2b43d2fa01c743be1b9273ab1eaf6c233df078d705",
+        urls = ["https://github.com/bazelbuild/rules_go/releases/download/0.16.5/rules_go-0.16.5.tar.gz"],
+    )
+
+
+def bazel_gazelle_deps():
```

Comment:
I think you need to pull my previous changes and merge them, since I ordered all functions by name, and this commit reverts that order for no real reason.